angular/projects/spark-angular/src/lib/directives/sprk-spinner/sprk-spinner.directive.ts

```diff
@@ -0,0 +1,33 @@
+import { AfterViewInit } from '@angular/core';
+import { Input } from '@angular/core';
+import { Directive, ElementRef, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: '[sprkSpinner]',
+})
+export class SprkSpinnerDirective implements AfterViewInit {
+  /**
+   * @ignore
+   */
+  constructor(private renderer: Renderer2, private el: ElementRef) {}
+
+  /**
+   *  Determines the spinner style.
+   */
+  @Input() variant: 'primary' | 'secondary' | 'dark';
```

Comment:
We need idString and addtionalClasses inputs here.

Reply:
✅